wagtail/admin/templates/wagtailadmin/panels/publishing/schedule_publishing_panel.html

```diff
@@ -13,7 +16,7 @@
     {% endif %}
 {% endif %}
 
-{% dialog id='schedule-publishing-dialog' dialog_root_selector='[data-edit-form]' classname=classname icon_name='calendar-alt' title=schedule_publishing_dialog_title subtitle=schedule_publishing_dialog_subtitle message_icon_name='info' message_status='info' message_heading=message_heading message_description=message_description %}
+{% dialog id='schedule-publishing-dialog' dialog_root_selector='[data-edit-form]' classname=classname icon_name='calendar-alt' title=schedule_publishing_dialog_title subtitle=schedule_publishing_dialog_subtitle|capfirst message_icon_name='info' message_status='info' message_heading=message_heading message_description=message_description %}
```

Comment:
`capfirst` in case the translation begins with `model_name`

Reply:
And here I have been avoiding putting the model name first in the Dutch translations because of capitalisation issues.

The translation teams thanks you!